changelogs/current.yaml

```
@@ -104,6 +104,10 @@ new_features:
 - area: access_log
   change: |
     added support for number values in substitution format string in json_format.
+- area: http
+  change: |
+    added the expected :ref:`receive <envoy_v3_api_field_config.core.v3.HealthCheck.HttpHealthCheck.receive>` payload check for HTTP health check.
+    added :ref:`response_buffer_size <envoy_v3_api_field_config.core.v3.HealthCheck.HttpHealthCheck.response_buffer_size>` to configure the maximum HTTP response buffer size.
```

Apologies, one small nit after looking at the rendered docs (just realized that these are the same entry in the version history).
```suggestion
    added the expected :ref:`receive <envoy_v3_api_field_config.core.v3.HealthCheck.HttpHealthCheck.receive>` payload check for HTTP health check.
    Added :ref:`response_buffer_size <envoy_v3_api_field_config.core.v3.HealthCheck.HttpHealthCheck.response_buffer_size>` to configure the maximum HTTP health check response buffer size.
```